bot/bot.py

```diff
@@ -0,0 +1,15 @@
+import logging
+
+from aiogram import Bot, Dispatcher, executor, types
+
+logging.basicConfig(level=logging.INFO)
+
+bot = Bot(token = '?')
+dp = Dispatcher(bot)
+
+@dp.message_handler()
+async def echo(message: types.Message):
+  await message.answer(message.text)
```

Comment:
**PEP 8: E111 indentation is not a multiple of four**
What does it means? Read [here](https://www.flake8rules.com/rules/E111.html)
> It can show in your editor normal, but look on GitHub there is identation of 2 spaces